### PR TITLE
Modified to handle errors during read path

### PR DESCRIPTION
**Pull request by parekuti**
Related to issue: No error is displayed when cassandra nodes are down for read path #120

### CODE REVIEW

File: cassandra/src/main/scala/filodb.cassandra/Util.scala [hunk 1]
```diff
@@ -20,6 +20,7 @@ object Util {
         if (resultSet.wasApplied) Success else notAppliedResponse
       }.recover {
         case e: DriverException => throw StorageEngineException(e)
+        case e: Exception       => throw e
```
Comment (velvia) [inline cassandra/src/main/scala/filodb.cassandra/Util.scala]: I think we should still wrap the error in StorageEngineException.

File: cassandra/src/main/scala/filodb.cassandra/Util.scala [hunk 2]
```diff
@@ -30,6 +31,7 @@ object Util {
       // from invalid Enum strings, which should never happen, or some other parsing error
       case e: NoSuchElementException   => throw MetadataException(e)
       case e: IllegalArgumentException => throw MetadataException(e)
+      case e: Exception                => throw e
```
Comment (velvia) [inline cassandra/src/main/scala/filodb.cassandra/Util.scala]: Same here, wrap in StorageEngineException

**Comment by parekuti** [comment]
Added this general exception to handle just in case of other errors than DriverException. But this is not the code actually solving the issue. The changes made to the ChunkRowMap table helped to throw exception back to the client.

File: cassandra/src/main/scala/filodb.cassandra/Util.scala (hunk 2 above)
Comment (parekuti) [inline cassandra/src/main/scala/filodb.cassandra/Util.scala]: Done

File: cassandra/src/main/scala/filodb.cassandra/Util.scala (hunk 1 above)
Comment (parekuti) [inline cassandra/src/main/scala/filodb.cassandra/Util.scala]: Done.

**Pull request merged**